webapp/components/signup/signup_controller.jsx

```diff
@@ -79,14 +79,14 @@ export default class SignupController extends React.Component {
                             }
                         );
                     },
-                    () => {
+                    (e) => {
                         this.setState({ // eslint-disable-line react/no-did-mount-set-state
                             noOpenServerError: true,
                             loading: false,
                             serverError: (
                                 <FormattedMessage
-                                    id='signup_user_completed.invalid_invite'
-                                    defaultMessage='The invite link was invalid.  Please speak with your Administrator to receive an invitation.'
+                                    id={e.id}
+                                    defaultMessage={e.message}
```

Comment:
Oh, I see what's happening. You're returning a server error with an ID that's not known on the client, so it defaults to the default message which is just the server error message that's returned. This will work but can you just remove the `FormattedMessage` element and just directly use `e.message`?